krun.py

```diff
@@ -210,7 +210,6 @@ def main(parser):
     except util.FatalKrunError as e:
         subject = "Fatal Krun Exception"
         mailer.send(subject, e.args[0], bypass_limiter=True)
-        util.run_shell_cmd_list(config.POST_EXECUTION_CMDS)
```

Comment:
This is the source of the duplicate call when a post-command fails, and `util.run_shell_command` calls `fatal()`. So, do you want to introduce a new exception type? `JustAFleshWoundKrunError`?